traffic_ops/traffic_ops_golang/wrappers.go

```diff
@@ -100,7 +100,7 @@ func wrapPanicRecover(h http.HandlerFunc) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		defer func() {
 			if err := recover(); err != nil {
-				log.Errorf("panic: (err: %v) stacktrace:\n%s\n", err, stacktrace())
+				api.HandleErr(w, r, nil, http.StatusInternalServerError, nil, fmt.Errorf("panic: (err: %v) stacktrace:\n%s\n", err, stacktrace()))
```

Comment:
Nitpick: a `return` after this would be safer. It doesn't matter right now, but if someone changed the func to do something else, they could miss that it needs to return after `HandleErr`. Safer to add it now.

Reply:
done